LICENSE.txt

```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2020, conda-forge contributors
```

Comment:
We should use the Anaconda license here I think.

Reply:
Fixed